WordPress/src/main/AndroidManifest.xml

```diff
@@ -66,8 +66,6 @@
         android:supportsRtl="true"
         tools:ignore="UnusedAttribute">
 
-        <meta-data android:name="io.sentry.auto-init" android:value="false" />
```

Comment:
This will finally end in merged AndroidManifest as Track's library [has it declared](https://github.com/Automattic/Automattic-Tracks-Android/blob/2.1.0/AutomatticTracks/src/main/AndroidManifest.xml#L9-L11).